subprojects/base-services/src/main/java/org/gradle/internal/classloader/TransformingClassLoader.java

```diff
@@ -83,7 +82,16 @@ public TransformingClassLoader(ClassLoader parent, Collection<URL> urls) {
     private byte[] loadBytecode(URL resource) throws IOException {
         InputStream inputStream = resource.openStream();
         try {
-            return ByteStreams.toByteArray(inputStream);
+            ByteArrayOutputStream out = new ByteArrayOutputStream();
```

Comment:
This change doesn't strictly belong here, but I didn't want to miss the opportunity. Eventually we want to isolate code that needs to run on Java 5 so we can upgrade to Guava 20. I think this code still needs to run on Java 5, so I removed Guava.